lms/djangoapps/grades/models.py

```diff
@@ -77,7 +79,7 @@ def to_json(self):
     @classmethod
     def from_json(cls, blockrecord_json):
         """
-        Return a BlockRecordSet from a json list.
+        Return a BlockRecordList from a previously serialized json.
```

Comment:
Will do.  I'll make this change as part of https://github.com/edx/edx-platform/pull/13440.
(So I don't have to wait for a full cycle on Jenkins.)